pkg/hubble/peer/handler.go

```diff
@@ -132,7 +134,7 @@ func (h *handler) Close() {
 // newChangeNotification creates a new change notification with the provided
 // information. If withTLS is true, the TLS field is populated with the server
 // name derived from the node and cluster names.
-func newChangeNotification(n types.Node, t peerpb.ChangeNotificationType, withTLS bool) *peerpb.ChangeNotification {
+func newChangeNotification(n types.Node, t peerpb.ChangeNotificationType, withTLS bool, preferIpv6 bool) *peerpb.ChangeNotification {
```

Comment:
nit: looks like we're providing most of the `handler` state as parameters to this `func`, so at that point making it a method could make sense

```suggestion
func (*h handler) newChangeNotification(n types.Node, t peerpb.ChangeNotificationType) *peerpb.ChangeNotification {
```